core/cbor_blake/ipld_version.hpp

```diff
@@ -23,4 +24,14 @@ namespace fc {
       return ipld->get(key);
     }
   };
+
+  // TODO(turuslan): refactor Ipld to CbIpld
+  inline IpldPtr withVersion(IpldPtr ipld, uint64_t height) {
+    const auto version{actorVersion(height)};
+    if (ipld->actor_version != version) {
```

Comment:
The default value of actor version is 0.
In case `version == 0` it does nothing.
Do you want to keep this inconsistent behaviour for different actor versions?

Reply:
This function fixes `Ipld.actor_version`.
If `Ipld` version does not match height, it's wrapped with corrent one.